src/qamqpqueue_p.h

```diff
@@ -55,6 +55,8 @@ class QAmqpQueuePrivate: public QAmqpChannelPrivate,
     bool consumeRequested;
 
     Q_DECLARE_PUBLIC(QAmqpQueue)
+    private:
+    QAmqpTable arguments;
```

Comment:
this is already the implementation for the dptr, no need to make this member private